packages/world/src/interfaces/IAccessManagementSystem.sol

```diff
@@ -10,5 +10,5 @@ interface IAccessManagementSystem {
 
   function revokeAccess(ResourceId resourceId, address grantee) external;
 
-  function transferOwnership(bytes14 namespace, address newOwner) external;
+  function transferOwnership(ResourceId namespaceId, address newOwner) external;
```

Comment:
We sorta talked about this in the other PR but I wonder if there's any value to having a `NamespaceId` wrapper rather than full resource ID where we mask/shift/slice the namespace from it.

Reply:
would prefer one user type for all of these because i want to add checks like `namespaceId.isType(RESOURCE_NAMESPACE)` in the next PR - if `namespaceId` uses a different user type, we'd have to duplicate the `ResourceId` library for it. (We can't trust the id provided as argument to actually be a valid namespace id, even if it would be of type `NamespaceId`, because anyone can just wrap whatever with `NamespaceId`)

Reply:
oh right, I forgot we have a namespace type, that makes sense